### ISSUE TITLE
(v3.0.0beta00) PyPAD Scripts Crash

### ISSUE THREAD

**Issue by deltecent** [ElvishArtisan/rivendell#353]
From ```pypad_filewrite.py```:
```
File "/usr/lib64/python3.4/json/decoder.py", line 359, in raw_decode
  obj.end = self.scan_once(s, idx)
ValueError: Invalid control character at line 60 column 39 (char 1837)
```
Based on the position in the log, this may have been caused by an ```&``` character.

I will test.

**Comment by deltecent** [comment]
No, not the ```&``` character...

**Comment by deltecent** [comment]
I cannot find anything in the Artist / Title on this system that would cause this error. It may be an issue with other PAD data. The JSON will have to be logged.

**Comment by ElvishArtisan** [comment]
'Control character' implies something in the range 0x00 - 0x1F. Do you know what specific cart provoked the crash? Might be worth looking for 'funny characters' in that metadata.

**Comment by deltecent** [comment]
We have not been able to track this down to any cart with any characters outside 0x20-0x7E. I also have not seen this problem on My production system. I am going to add some extra debugging to Floor104 and see what we see.

Again, maybe I shouldn't open an issue until I have 100% certainty to the cause of the problem.

**Comment by deltecent** [comment]
Should we have ```pypad.api``` handle syslog and provide a logging function so we can catch and log errors, such as JSON decoding errors, rather than have the entire PyPad script crash? If you have not noticed, I'm a big fan of syslog. For example:

```
if line.decode('utf-8')=="\r\n":
    try:
        self.__pypad_Process(Update(json.loads(msg),self.__config_parser))
    except json.JSONDecodeError(msg, doc, pos):
        syslog('pypad.py: error decoding json message ('+msg+') "'+doc+'"')
    msg=""
```

(Example code not tested...)

**Comment by deltecent** [comment]
Ok, found it. It ended up being the Next data that was causing the problem.

Question is, should having a "Control Character" in the metadata crash a PyPad script?

![screen shot 2019-01-21 at 4 59 29 pm](https://user-images.githubusercontent.com/24786756/51505928-57684400-1d9e-11e9-83b4-a429156250c8.png)

**Comment by ElvishArtisan** [comment]
Fixed in [master f8b223e].
